ui/src/app/views/account-creation/component.tsx

```diff
@@ -191,6 +201,10 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
                      value={contactEmail}
                        style={{width: '16rem'}}
                        onChange={v => this.updateProfile('contactEmail', v)}/>
+          {this.state.invalidEmail &&
+          <Error id='invalidEmailError'>
+            Email Id is invalid
```

Comment:
Nit: I would probably say `Contact Email is invalid`

Reply:
Error message has been updated and yes the regex is picked up online as standard for email structure check